### PR TITLE
Endpoints + EndpointPermissions

### LINKED ISSUE #629: Admin endpoints + endpoint permissions
With #120 , the basic admin module (with applications, async jobs, config, endpoints, roles) is available, in readonly mode.

Enhance endpoints view in Admin Endpoints page, adding "permissions" (this requires a new feature, that will be developed in BEdita/API [update: under work in https://github.com/bedita/bedita/pull/1908]).

### PR DESCRIPTION
**Pull request by didoda**
This fixes #629

### CODE REVIEW

File: src/Controller/Admin/AdministrationBaseController.php [hunk 1]
```diff
@@ -133,10 +133,19 @@ public function index(): ?Response
         $this->set('propertiesSecrets', $this->propertiesSecrets);
         $this->set('metaColumns', $this->meta);
         $this->set('filter', []);
-        $this->set('schema', (array)$this->Schema->getSchema($this->resourceType));
+        $this->set('schema', []);
         $this->set('readonly', $this->readonly);
         $this->set('deleteonly', $this->deleteonly);
 
+        if ($this->resourceType === 'endpoint_permissions') {
```
Comment (stefanorosanelli) [inline src/Controller/Admin/AdministrationBaseController.php]: We should probably add these lines in `EndpointPermissionsController::index()` ?

File: src/Controller/Admin/AdministrationBaseController.php [hunk 2]
```diff
@@ -133,10 +133,19 @@ public function index(): ?Response
         $this->set('propertiesSecrets', $this->propertiesSecrets);
         $this->set('metaColumns', $this->meta);
         $this->set('filter', []);
-        $this->set('schema', (array)$this->Schema->getSchema($this->resourceType));
+        $this->set('schema', []);
```
Comment (stefanorosanelli) [inline src/Controller/Admin/AdministrationBaseController.php]: are we sure it's never used?

Comment (didoda) [reply src/Controller/Admin/AdministrationBaseController.php]: admin endpoints lack schema (they are not objects). am I right?

Comment (stefanorosanelli) [reply src/Controller/Admin/AdministrationBaseController.php]: also resources have a schema => look in `config/schema_properties.php` - schema is fixed on resources and dynamic on objects

**Pull request merged**